plugins/Vestige/Vestige.cpp

```diff
@@ -429,8 +429,35 @@ bool VestigeInstrument::handleMidiEvent( const MidiEvent& event, const TimePos&
 	return true;
 }
 
+bool VestigeInstrument::presetChangeSupported()
+{
+	return true;
+}
 
-
+void VestigeInstrument::changePreset(int bank, unsigned int preset)
+{
+	if (m_plugin)
```

Comment:
Can we invert the guard to return early and not double indent the whole function?

Reply:
done